core/src/main/java/greencity/configuration/SecurityConfig.java

```diff
@@ -72,7 +72,7 @@ public PasswordEncoder passwordEncoder() {
      *
      * @param http {@link HttpSecurity}
      */
-
+    @SuppressWarnings("java:S4502")
```

Comment:
@SuppressWarnings("squid:S4502")

Reply:
This works as well due to https://www.baeldung.com/sonar-exclude-violations